app/src/components/SingleDatePicker/SingleDatePicker.js

```diff
@@ -0,0 +1,113 @@
+import React, { useCallback, useMemo, useRef, useState } from 'react'
+import PropTypes from 'prop-types'
+import { Popover, GU, RADIUS, useTheme } from '@aragon/ui'
+import DatePicker from './DatePicker'
+import Labels from './Labels'
+import { SINGLE_DATE } from './consts'
+import { dayjs, dateFormat } from '../../utils/date-utils'
+import handleSingleDateSelect from './utils'
+
+function SingleDatePicker({ format, onChange, startDate: startDateProp }) {
```

Comment:
Perhaps now since we are not using a local state we can just use `startDate` here?

```suggestion
function SingleDatePicker({ format, onChange, startDate }) {
```